keylime/cloud_verifier_common.py

```diff
@@ -114,7 +115,10 @@ def init_mtls(section='cloud_verifier', generatedir='cv_ca'):
 
     try:
         context = ssl.create_default_context(ssl.Purpose.CLIENT_AUTH)
-        context.minimum_version = ssl.TLSVersion.TLSv1_2
+        if sys.version_info >= (3,7):
+            context.minimum_version = ssl.TLSVersion.TLSv1_2
+        else:
+            context.options &= ~ssl.OP_NO_TLSv1_2
```

Comment:
For keylime maintainers: I am wondering if we cannot default/target to Python 3.6 so we don't have these inconsistencies?

Reply:
Not sure if it is a viable strategy, giving the uses in production.

Reply:
Yeah I agree with @maugustosilva we can't restrict it to sub versions of python. We can do things like requiring a minimum version, but not a specific version.